src/content/extensions/multipleroutes/src/_locales/en/messages.json

```diff
@@ -0,0 +1,23 @@
+{
+   "NETLI_MULTIROUTES_CHECKBOX": {
+      "message": "Enables the WebRTC multiple-routes option in Chrome's privacy settings."
+   },
+   "NETLI_MULTIROUTES_DESCRIPTION": {
+      "message": "If enabled, Chrome will explore all possible routing options when using WebRTC to find the most performant path, possibly exposing user's private IP address. Otherwise, WebRTC traffic will be routed the same way as regular HTTP."
```

Comment:
"routed the same way as regular HTTP" => "will be routed through the default network route, similar to HTTP traffic".